audit/audit-gcp.sh

```diff
@@ -98,6 +98,31 @@ gcloud \
     | sed 's/.googleapis.com//' \
     | while read -r SVC; do
         case "${SVC}" in
+            bigquery)
+                mkdir -p "projects/${PROJECT}/services/${SVC}"
+                rm -f projects/${PROJECT}/services/${SVC}/bigquery.datasets.* || :
+                bq \
+                    --format=json --project_id=$PROJECT ls
+                    > "projects/${PROJECT}/services/${SVC}/bigquery.datasets.json"
+                
+                # Only run if there are any datasets
+                if [ -s "projects/${PROJECT}/services/${SVC}/bigquery.datasets.json" ]
+                then
+                    bq \
+                        --project_id="{$PROJECT}" --format=json ls \
+                        | jq -r '.[] | .datasetReference["datasetId"]' \
+                        | while read -r DATASET; do                        
+                            bq \
+                                --project_id="${PROJECT}" --format=json show "${PROJECT}:${DATASET}" \
+                            | jq .access > "projects/${PROJECT}/services/${SVC}/bigquery.datasets.access.json"
+                        done
+                fi
+                gcloud \
```

Comment:
This is cute.  We don't do it elsewhere and it is duplicative of the overall IAM dumps above - do you think it's notably better?

Reply:
This was a leftover from the previous version , before I had to rebase. I though that every service would store its iam data. Removed.